app/experimenter/static/css/experimenter.css

```diff
@@ -216,3 +216,11 @@ pre.json {
   background-color: white;
   border-color: #ced4da;
 }
+
+.required-label.required .text-muted {
+  display: none;
+}
+
+.required-label.optional .text-danger {
+  display: none;
+}
```

Comment:
I added this little thing so we can flip 'optional' and 'required' fields in static forms easily.